linux.sh

```diff
@@ -0,0 +1,220 @@
+#!/bin/sh
+
+
+# read the env type and suite name from commandline arguments
+# the env_type can be "on-prem" or "remote"
+# set on-prem to run suite locally, and remote to run on suite browserstack
+# set suite name, we have defined different test combinations depending on suite name which cover 
+# a broad range of usecases
+env_type=$1
+suite=$2
+
```

Comment:
Would windows users be able to run the tests with sh?

Reply:
yes, they would simply need to install git [stack overflow link](https://stackoverflow.com/questions/26522789/how-to-run-sh-on-windows-command-prompt)

later we can also at a bat file

Reply:
Since you are working on the readme can u add in the readme

Reply:
Can yo add a comment with profile name for every profile in the runner.sh file